docs/code/IDataViewTypeSystem.md

```diff
@@ -661,154 +605,99 @@ This chapter contains information on the C# classes used to represent column
 types. Since the IDataView type system is extensible this list describes only
 the core data types.
 
-### `ColumnType` Abstract Class
+### `DataViewType` Abstract Class
 
-The IDataView system includes the abstract class `ColumnType`. This is the
-base class for all column types. `ColumnType` has several convenience
+The IDataView system includes the abstract class `DataViewType`. This is the
+base class for all column types. `DataViewType` has several convenience
```

Comment:
`base class for all column types.`  => `base class for all types used in the IDataView system.`?

Reply:
Right, right -- I see a few more places where we are still using the phrase "column type."